ConsoleApp3/ConsoleApp3/Program.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+
+namespace ConsoleApp
+{
+
+    public class Stack<T>
+    {
+        public T[] Stck { get; set; }
+        public int Top { get; set; }
+        public int Size { get; }
+
+        public Stack(int Size)
+        {
+            Stck = new T[Size]; 
+            this.Size = Stck.Length;
+        }
+
+        public void Push(T Element)
+        {
+
+            if (Top == Size)
+            {
+                throw new InvalidOperationException("You are nerd!!! STACK OVERFLOW!!!");
+            }
+            else
+            {
+            Stck[Top] = Element;
```

Comment:
of course it will work, but stack should be dynamic structure and in case if somebody want to store more elements - he should be able